DEVELOPMENT.md

```diff
@@ -202,3 +202,48 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
+
+For that, you'll need to create the following configurations: 
+
+### Worker
+
+First you'll have to set a name. We recommend using `worker`.
+
+Then you'll have to add the following config:
+
+```
+Ruby script: /Users/lvillenj/Projects/apisonator/bin/3scale_backend_worker
+Script arguments: --debug
+Working directory: /Users/lvillenj/Projects/apisonator
+Environment variables: RACK_ENV=development
+```
+
+### Puma Listener
+
+First you'll have to set a name. We recommend using `listener (puma)`.
+
+Then you'll have to add the following config:
+
+```
+Ruby script: /Users/lvillenj/Projects/apisonator/bin/3scale_backend
+Script arguments: -s puma -X "-w 0" start -p 3001
+Working directory: /Users/lvillenj/Projects/apisonator
+Environment variables: RACK_ENV=development;LISTENER_WORKERS=1
```

Comment:
Does `LISTENER_WORKERS` do anything? I think it doesn't change the behavior for me, `"-w 0"` already sets puma server in "single mode", so there are actually no additional workers created. I think I'd remove it.

Reply:
IMO it does nothing